spec/support/statistics_helpers.rb

```diff
@@ -1,27 +1,45 @@
 module StatisticsHelpers
-  def performance_platform_response_for_apply_uk_visa
+  def performance_platform_response
     {
-        search_terms: performance_platform_response_for_search_terms,
-        searches: performance_platform_response_for_searches,
-        page_views: performance_platform_response_for_page_views,
-        problem_reports: performance_platform_response_for_problem_reports
+        search_terms: response_for_search_terms,
+        searches: response_for_searches,
+        page_views: response_for_page_views,
+        problem_reports: response_for_problem_reports
+    }
+  end
+
+  def performance_platform_response_with_nil_values
+    {
+        search_terms: response_for_search_terms_with_nil_values,
+        searches: response_for_searches_with_nil_values,
+        page_views: response_for_page_views_with_nil_values,
+        problem_reports: response_for_problem_reports_with_nil_values
     }
   end
 
   def performance_platform_response_for_multipart_artefact
     {
-        search_terms: performance_platform_response_for_search_terms,
-        searches: performance_platform_response_for_multipart_searches,
-        page_views: performance_platform_response_for_multipart_page_views,
-        problem_reports: performance_platform_response_for_multipart_problem_reports
+        search_terms: response_for_search_terms,
+        searches: response_for_multipart_searches,
+        page_views: response_for_multipart_page_views,
+        problem_reports: response_for_multipart_problem_reports
+    }
+  end
+
+  def performance_platform_response_for_multipart_with_nil_values
+    {
+        search_terms: response_for_search_terms_with_nil_values,
+        searches: response_for_multipart_searches_with_nil_values,
+        page_views: response_for_multipart_page_views_with_nil_values,
+        problem_reports: response_for_multipart_problem_reports_with_nil_values
     }
   end
 
   def performance_platform_response_with_no_performance_data
     {}
   end
 
-  def performance_platform_response_for_searches
+  def response_for_searches
```

Comment:
I'm a bit down on the method name changes here - they were more explicit before which I think is better.  By renaming them we also introduce a bunch of noise to the diff that we have to review that doesn't really add anything.  If we leave them as-is the diff is smaller and your actual change is clearer.